avm/utilities/pipelines/platform/Set-AvmGitHubIssueOwnerConfig.ps1

```diff
@@ -131,8 +131,8 @@ A member of the @azure/$($module.ModuleOwnersGHTeam) or @azure/$($module.ModuleC
     }
 
     if ([String]::IsNullOrEmpty($assign)) {
-      $reply = "This issue couldn't be assigend to $($module.PrimaryModuleOwnerGHHandle), because no such users exists."
       if ($PSCmdlet.ShouldProcess("missing user comment to issue [$($issue.title)]", 'Add')) {
+        $reply = "This issue couldn't be assigend due to an internal error. @$($module.PrimaryModuleOwnerGHHandle), please make sure this issue is assigned to you and please provide an initial response within 5 business days."
```

Comment:
Any chance to change this to:

"...and please provide an initial response as soon as possible, in accordance with our [Support statement](https://aka.ms/AVM/Support)."